.travis.yml

```diff
@@ -69,6 +69,7 @@ install:
         pip install -r requirements.txt
         pip install seaborn sphinx==1.4.9 pytest pytest-cov
       fi
+    - pip install testfixtures
```

Comment:
maybe try `pip install --upgrade --no-deps testfixtures`, I noticed on the 3.4 build it says it was already installed so this line did nothing

Reply:
actually that was on the 2.7 build

BTW the 2.7 build still has the old `len(context)` error. You could try reproducing locally with a `conda` env, it probably has to do with `sphinx=1.4.9`

Reply:
I don't remember how the `len(context)` error got resolved...was it just by pinning a different version of sphinx? Is that a change I should add to this PR?

Reply:
just added the `--upgrade` but that didn't work...also I added a python 3.4 environment on my computer and the tests pass there, so I dunno why it isn't working for 3.4 in travis...

Reply:
On a Linux system, or OSX? It might need to match the OS to ensure conda is pulling in the same packages.

If you don't have access to a Linux system I can give it a try

Reply:
I'm operating on Ubuntu right now (well...it's complicated because I'm technically workin on Windows Subsystem for Linux, which is linux running on Windows 10, but for the purposes of conda it's ubuntu)